### ISSUE TITLE
Classification of crystallization outcomes using deep convolutional neural networks

### ISSUE THREAD

**Issue by stephenra** [greenelab/deep-review#856]
>The Machine Recognition of Crystallization Outcomes (MARCO) initiative has assembled roughly half a million annotated images of macromolecular crystallization experiments from various sources and setups. Here, state-of-the-art machine learning algorithms are trained and tested on different parts of this data set. We find that more than 94% of the test images can be correctly labeled, irrespective of their experimental origin. Because crystal recognition is key to high-density screening and the systematic analysis of crystallization experiments, this approach opens the door to both industrial and fundamental research applications.
https://arxiv.org/abs/1803.10342

**Comment by Johnny65456** [comment]
Hi stephenra, about the paper, I'm curious about the weight decay of 0.9. Because it is really a huge decay, which means for each epoch, only the few starting iterations have larger learning rate.
